app/uk/gov/hmrc/nationaldutyrepaymentcenter/controllers/ClaimController.scala

```diff
@@ -80,6 +87,9 @@ class ClaimController @Inject() (
                     )
                   )
                 )
+                logger.info(
+                  s"CreateCaseEvent failed with bad request, caseReferenceNumber:[] correlationId:[$correlationId]"
```

Comment:
There's no case ref number here - should there be?

Reply:
No - on a CreateCaseEvent failure, PEGA never gave us a case ref number, so all we can do is error and show that fact.